### ISSUE TITLE
Not compiling under 2.0.10 and *.rc issue under *nix

### ISSUE THREAD

**Issue by gcarreno** [wp-xyz/corona#30]
Hey Werner(@wp-xyz),

#### Compile error under 2.0.10
As of
> 367ecd6 (HEAD -> mapping, origin/mapping) Plotting of world map. No connection to Covid data, yet.

Lazarus stable is complaining:
```console
$ lazbuild source/corona.lpi
# ...
/home/gcarreno/Programming/corona/source/cpolygonseries.pas(188,43) Error: (3026) Wrong number of parameters specified for call to "GetLegendItemsRect"
/home/gcarreno/FreePascal_stable/lazarus/components/tachart/lib/x86_64-linux/gtk2/tacustomseries.ppu:tacustomseries.pas(1464,29) Error: (5088) Found declaration: GetLegendItemsRect(TChartLegendItems;TBrush);
cpolygonseries.pas(245) Fatal: (10026) There were 2 errors compiling module, stopping
# ...
```
#### The new corona_resources.rc

Lazarus uses `windres` to compile the `*.rc` files.
I found `i686-w64-mingw32-windres` on Ubuntu package `binutils-mingw-w64-i686`.
I installed it and then had to do a `ln -s i686-w64-mingw32-windres windres` for lazbuild trunk to compile with success.

I think we should consider `.lrs`. It's the same thing and, to be honest, no external dependencies in terms of resource compiler because Lazarus has all the tools to manipulate `*.lrs`.

Just my 2&cent;s

**EDIT**: have a look at [Lazarus resources](https://wiki.freepascal.org/Lazarus_Resources#Lazarus_resources)

Cheers,
Gus

**Comment by gcarreno** [comment]
After reading [Lazarus Resources: Checking you have windres](https://wiki.freepascal.org/Lazarus_Resources#Checking_you_have_windres) I now know that I don't need to 
```console
$ sudo ln -s i686-w64-mingw32-windres windres
```
I only have to install the package I mentioned and then mess with `fpc.cfg`, YAY!!

**Comment by gcarreno** [comment]
Closed by 7537c42 and c751c9e
